interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -510,7 +510,7 @@ MathFuncts, errGen) {
   function SampleBinCtrl($scope, SampleBin, MlModelTracker) {
     // give our templates a way to access the SampleBin service
     $scope.sb = SampleBin;
-    $scope.modelInfo = MlModelTracker;
+    SampleBin.mlModelInfo = MlModelTracker;
```

Comment:
I know that this works and does not really make a difference in terms of functionality, but would it make sense to change the order of these two statements like this:
```
SampleBin.mlModelInfo = MlModelTracker;
$scope.sb = SampleBin;
```
?
I feel like this makes more sense intuitively since we are setting the `mlModelInfo` property to `MlModelTracker` first and *then* assigning $scope.sb to this, which will let the page access this information.